### ISSUE TITLE
`fpt script` command not working as expected

### ISSUE THREAD

**Issue by srpomeroy** [rightscale/policy_sdk#9]
Following the tutorial here: https://docs.rightscale.com/policies/getting_started/policy_tutorial.html#processing-data

`fpt script tutorial.pt addresses=@mock_addresses.json ip_whitelist=[]`

The expected result is output.json should contain an array with two objects but it is instead null.
I added a console.out as the very last step of the javascript block and see that unattached_ips does indeed contain those two objects.

Tested in Windows 10 and Ubuntu 16.04 with the same result.

**On Ubuntu:**
```
spomeroy@laptop: /fpt$ ./fpt script /mnt/c/users/spomeroy/downloads/tutorial.pt addresses=@/mnt/c/users/spomeroy/downloads/mock_addresses.json ip_whitelist=[]
Running script "js_munge_addresses" from /mnt/c/users/spomeroy/downloads/tutorial.pt and writing unattached_ips" to out.json
console.log: "iterating over addresses, count:" 2
console.log: >
 "unattached_ips result: " [
    {
      "address": "10.20.30.50",
      "attached": false,
      "excluded": false,
      "href": "/api/clouds/1/ip_addresses/B41PVJI70FMLO",
      "name": "unattached_ip_address"
    },
    {
      "address": "10.20.30.40",
      "attached": true,
      "excluded": false,
      "href": "/api/clouds/1/ip_addresses/4C25RVT8G7U3B",
      "name": "attached_ip_address"
    }
  ]
JavaScript finished, duration=8.8919ms
spomeroy@laptop:~/fpt$ cat out.json
null
```

**On Windows:**
```
C:\Users\SPomeroy\Downloads> fpt script .\tutorial.pt addresses=@mock_addresses.json ip_whitelist=[]
Running script "js_munge_addresses" from .\tutorial.pt and writing unattached_ips" to out.json
console.log: "iterating over addresses, count:" 2
console.log: >
 "unattached_ips result: " [
    {
      "address": "10.20.30.50",
      "attached": false,
      "excluded": false,
      "href": "/api/clouds/1/ip_addresses/B41PVJI70FMLO",
      "name": "unattached_ip_address"
    },
    {
      "address": "10.20.30.40",
      "attached": true,
      "excluded": false,
      "href": "/api/clouds/1/ip_addresses/4C25RVT8G7U3B",
      "name": "attached_ip_address"
    }
  ]
JavaScript finished, duration=997.4µs
C:\Users\SPomeroy\Downloads> get-content out.json
null
```

[policy_tutorial.zip](https://github.com/rightscale/policy_sdk/files/3478470/policy_tutorial.zip)

**Comment by srpomeroy** [comment]
**On Ubuntu:**
fpt v1.0.3 - 2019-03-13 23:54:04 - f3993acac5a7a52b3c9848aba610c6860db23017

**On Windows:** 
fpt v1.0.3 - 2019-03-13 23:54:39 - f3993acac5a7a52b3c9848aba610c6860db23017

**Comment by psschroeter** [comment]
Running dos2unix on the tutorial.pt fixes the issue for me. The script command does a bunch of regex's internally to parse out parameters and such -- should be easy enough to fix by normalizing the newlines.

**Comment by rlafleur** [comment]
> Running dos2unix on the tutorial.pt fixes the issue for me. The script command does a bunch of regex's internally to parse out parameters and such -- should be easy enough to fix by normalizing the newlines.

Verified workaround on my Windows 10 laptop. Thanks!

**Comment by douglaswth** [comment]
This should be fixed by #10 and released in `v1.0.41.
